packages/TypeDeclaration/tests/Rector/Property/PropertyTypeDeclarationRector/Fixture/doctrine_relation.php.inc

```diff
@@ -44,6 +57,12 @@ class DoctrineRelation
      */
     private $products;
 
+    /**
+     * @ORM\OneToOne(targetEntity="DoctrineRelated")
+     * @var DoctrineRelated
```

Comment:
```suggestion
     * @var \Rector\TypeDeclaration\Tests\Rector\FunctionLike\PropertyTypeDeclarationRector\Fixture\DoctrineRelated
```

This should be FQN by default, that's convention for all changed imports here to have a consistent output